Extensions/better_reblogs.js

```diff
@@ -0,0 +1,264 @@
+        }
+    },
+
+    run: function() {
+        this.running = true;
+
+        if (this.preferences.type.value === "nested"){
+            this.run_nested();
+        } else {
+            this.run_flat();
+        }
+
+    },
+
+    frame_run: function() {
+        if (XKit.page.peepr) {
+            XKit.extensions.better_reblogs.run();
+        }
+    },
+
+    run_flat: function() {
+
+        var list_sel = "";
+        if (this.preferences.remove_last_user.value){
+            XKit.tools.add_css(".reblog-list-item.contributed-content .reblog-header {display: none;}", "better_reblogs");
+            list_sel = ".reblog-list ";
+        }
+
+        if (this.preferences.margin.value) {
+            XKit.tools.add_css(list_sel+".reblog-list-item .reblog-content {margin-left:35px;}" +
+                               list_sel+".reblog-list-item .reblog-title {margin-left:35px;}", "better_reblogs");
+        }
+
+        if (this.preferences.remove_icon.value) {
+            XKit.tools.add_css(".reblog-header .sub-icon-reblog:before {display: none!important;}" +
+                               ".reblog-header .sub-icon-reblog:after {display:none!important;}", "better_reblogs");
+
+        }
+
+        if (this.preferences.add_border.value) {
+            XKit.tools.add_css(list_sel+".reblog-list-item .reblog-content {border-left: 2px solid #E7E7E7; padding-left: 10px;}" +
+                               ".post.post_full "+list_sel+".reblog-list-item .tmblr-full > img {padding: 0 20px}", "better_reblogs");
+
+            if (!(this.preferences.margin.value || this.preferences.remove_user_names.value)) {
+                XKit.tools.add_css(".reblog-list-item .reblog-content {margin-left: 3px;}", "better_reblogs");
+            }
+        }
+
+        if (this.preferences.remove_user_names.value) {
+            XKit.tools.add_css(".reblog-tumblelog-name {display:none;} .reblog-list-item .reblog-header {margin-bottom: 0;} "+
+                               list_sel+".reblog-content {margin-left:35px;} .reblog-title {margin-left:35px; margin-top:-10px;}", "better_reblogs");
+        }
+
+        if (this.preferences.remove_avatars.value) {
+            XKit.tools.add_css(".reblog-avatar {display:none !important;} .reblog-header {padding-left: 0px !important;}",
+                "better_reblogs");
+        }
+
+        if (this.preferences.slim_new_reblog.value) {
+            XKit.tools.add_css(".reblog-list-item {padding: 10px 20px 5px !important; min-height: 41px;}",
+                "better_reblogs");
+        }
+
+        if (this.preferences.reorder_reblog_title.value) {
+            XKit.tools.add_css(".reblog-list-item .reblog-title {margin-left:0!important;}", "better_reblogs");
+        }
+
+        XKit.extensions.better_reblogs.do_flat();
+        XKit.post_listener.add("better_reblogs", XKit.extensions.better_reblogs.do_flat);
+
+    },
+
+    run_nested: function() {
+        var match_only_main_dash = "ol" ? !XKit.page.peepr : "";
```

Comment:
I think this should be ``var match_only_main_dash = !XKit.page.peepr ? "ol" : "";``